### PR TITLE
Resiliency module support by CSM-Operator

### PR DESCRIPTION
**Pull request by adarsh-dell**
# Description
Adding support of resiliency module for supported drivers by CSM-Operator.

# GitHub Issues
List the GitHub issues impacted by this PR:

| GitHub Issue # |
| -------------- |
|https://github.com/dell/csm/issues/739|
|https://github.com/dell/csm/issues/785|

# Checklist:

- [x] Have you run a grammar and spell checks against your submission?
- [x] Have you tested the changes locally?
- [x] Have you tested whether the hyperlinks are working properly?
- [x] Did you add the examples wherever applicable?
- [ ] Have you added high-resolution images?
image
image

### CODE REVIEW

File: content/docs/deployment/csmoperator/modules/resiliency.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+linkTitle: "Resiliency"
+description: >
+  Pre-requisite for Installing Resiliency via Dell CSM Operator
+---
+
+The CSM Resiliency module for supported Dell CSI Drivers can be installed via the Dell CSM Operator. Dell CSM Operator will deploy CSM Resiliency sidecar.
+
+## Prerequisite
+
+The first thing to remember about _CSM for Resiliency_ is that it only takes action on pods configured with the designated label. Both the key and the value have to match what is in the resiliency module configuration. CSM for Resiliency emits a log message at startup with the label key and value it is using to monitor pods:
```
Comment (shanmydell) [inline content/docs/deployment/csmoperator/modules/resiliency.md:12]: rephrase this sentence

**Review by shanmydell — CHANGES_REQUESTED** [review]

File: content/docs/deployment/csmoperator/modules/resiliency.md (hunk 1 above)
Comment (adarsh-dell) [reply content/docs/deployment/csmoperator/modules/resiliency.md:12]: How about this one?

image

**Review by shanmydell — APPROVED** [review]

File: content/docs/deployment/csmoperator/modules/resiliency.md (hunk 1 above)
Comment (adarsh-dell) [reply content/docs/deployment/csmoperator/modules/resiliency.md]: Fixed it as suggested.

**Review by suryagupta4 — APPROVED** [review]

**Review by rajkumar-palani — APPROVED** [review]
lgtm

**Pull request merged**